app/src/main/java/edu/uc/groupProject/topten/ui/main/MainViewModel.kt

```diff
@@ -41,5 +46,24 @@ class MainViewModel : ViewModel() {
         return service.getStrawpoll(id)
     }
 
+    fun setIncrementTime(countdownTime: Long) {
```

Comment:
These are functions I added to provide functions to be used by the Fragemnts instead of directly using the Service.